### DISCUSSION TITLE
Support for OSS stack

### DISCUSSION THREAD

**Discussion by srinidhis94** [Grokzen/docker-redis-cluster#172] [General]
Hey,

Any plans to add support for redis oss stack ? 

https://github.com/Grokzen/docker-redis-cluster/discussions

**Comment by Grokzen** [comment]
@srinidhis94 No, what i am leaning towards right now is to not support anything above 7.2 due to the license debacle and fork this repo and make a new version based on Valkey and support that one moving forward. This might change at any time. But right now i will not build and publish anything above 7.2 and that probably includes OSS stack
